scipy/_lib/_disjoint_set.py

```diff
@@ -0,0 +1,175 @@
+"""
+Disjoint set data structure
+"""
+import collections
+
+
+class DisjointSet:
+    """ Disjoint set data structure for incremental connectivity queries.
+
+    .. versionadded: 1.6.0
+
+    Attributes
+    ----------
+    n_subsets : int
+        The number of subsets.
+
+    Methods
+    -------
+    add
+    merge
+    connected
+    __getitem__
+
+    Notes
+    -----
+    This class implements the disjoint set [1]_, also known as the *union-find*
+    or *merge-find* data structure. The *find* operation (implemented in
+    `__getitem__`)* implements the *path halving* variant. The *merge* method
```

Comment:
Is the first asterisk supposed to be there? Renders as follows in the docs:

![image](https://user-images.githubusercontent.com/6169306/92332581-40fc2280-f07f-11ea-941b-b82115a23ca3.png)

Reply:
Whoops, no.
Fixed.